app/tests/spec/models/auth_brokers/fx-desktop-v2.js

```diff
@@ -73,12 +73,22 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('afterForceAuth', function () {
+      it('notifies the channel with `fxaccounts:login`, halts', function () {
```

Comment:
`by default` didn't add any value to any of those messages, so I removed them all.